lib/galaxy/jobs/__init__.py

```diff
@@ -44,7 +45,9 @@
 TOOL_PROVIDED_JOB_METADATA_FILE = 'galaxy.json'
 
 # Override with config.default_job_shell.
-DEFAULT_JOB_SHELL = '/bin/sh'
+DEFAULT_JOB_SHELL = '/bin/bash'
+
+DEFAULT_CLEANUP_JOB = "always"
```

Comment:
John shouldn’t we use https://github.com/jmchilton/galaxy/blob/16.04_jobs/config/galaxy.ini.sample#L1064 or is this needed to galaxy-lib?
It can be confusing if we have the default parameters spread around multiple files.